rid/nn_test/constants.py

```diff
@@ -0,0 +1,54 @@
+# rid task name format
```

Comment:
why is there a `constants.py` at sub-directory `nn_test`, is it different from the `constants.py` at the main directory or separately maintained?.